runtime/autoload/man.vim

```diff
@@ -1,6 +1,11 @@
-let s:man_tag_depth = 0
+" Ensure Vim is not recursively invoked (man-db does this)
+" More info here http://comments.gmane.org/gmane.editors.vim.devel/29085
+let s:man_cmd = 'unset MANPAGER; man 2>/dev/null '
```

Comment:
Can we please wait on this change until https://github.com/neovim/neovim/issues/3265 is implemented?